### PR TITLE
Add support for Coverage resourceType

### PR DESCRIPTION
**Pull request by atolivero**
Do this to enable downstream changes to lifeomic-insights-etl

http://hl7.org/fhir/STU3/Coverage.html

### CODE REVIEW

File: src/main/scala/com/lifeomic/fhirlib/v3/resources/Coverage.scala [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                val subscriber: Option[Reference],
+                val subscriberId: Option[String],
+                val beneficiary: Option[Reference],
+                val relationship: Option[CodeableConcept],
+                val period: Option[Period],
+                val payor: Option[List[Reference]],
+                val grouping: Option[Grouping],
+                val dependent: Option[String],
+                val sequence: Option[String],
+                val order: Option[Integer],
```
Comment (atolivero) [inline src/main/scala/com/lifeomic/fhirlib/v3/resources/Coverage.scala:43]: I opted for using a base `Integer` type here, but the FHIR docs outline a "positiveInt" type.  Should that be added as a class in `datatypes/`?

Comment (mjtieman) [reply src/main/scala/com/lifeomic/fhirlib/v3/resources/Coverage.scala:43]: Doing type refinement seems heavy handed, `Integer` is ok.

**Comment by mjtieman** [comment]
Need a minor version bump

**Comment by atolivero** [comment]
@mjtieman Done (squashed it into the single commit)

**Review by mjtieman — APPROVED** [review]

**Review by simons5593 — APPROVED** [review]

**Pull request merged**